android/engine/src/main/java/org/smartregister/fhircore/engine/util/extension/DateTimeExtension.kt

```diff
@@ -80,6 +80,11 @@ fun Date.plusYears(years: Int): Date {
 
 fun DateType.format(): String = SDF_YYYY_MM_DD.format(value)
 
+fun DateTimeType.format(): String =
+  SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ").format(value).let {
```

Comment:
can we add a unit test for this function?

Reply:
sure, trying now

Reply:
added